libvast/src/command.cpp

```diff
@@ -436,9 +436,9 @@ run(const command::invocation& invocation, caf::actor_system& sys,
   if (auto search_result = fact.find(invocation.full_name);
       search_result != fact.end()) {
     auto merged_invocation = invocation;
+    // The original `sys.config()` was merged with the invocation options
+    // by `init_config()` above before this function is called.
     merged_invocation.options = content(sys.config());
-    for (const auto& [key, value] : invocation.options)
-      merged_invocation.options.insert_or_assign(key, value);
     return std::invoke(search_result->second, merged_invocation, sys);
```

Comment:
This whole change can just be reduced to this:

```cpp
invocation.options = content(sys.config());
return std::invoke(search_result->second, invocation, sys);
```

Reply:
But `invocation` is const in this context.

Reply:
In that case it might just make sense to have the function take the parameter by value.

Reply:
As it turns out, we *do* need to merge after all. After the latest update, passing by const& or by value would return in the same number of lines changed.